cli/src/commands/move.rs

```diff
@@ -62,6 +62,14 @@ pub(crate) fn cmd_move(
     command: &CommandHelper,
     args: &MoveArgs,
 ) -> Result<(), CommandError> {
+    writeln!(
+        ui.warning(),
+        "warning: `jj move` is deprecated; use `jj squash` instead, which is equivalent"
+    )?;
+    writeln!(
+        ui.warning(),
+        "warning: `jj move` will be removed in a future version, and this will be a hard error"
```

Comment:
Will it be a hard error?  What about the possibility that `move` will exist to be like other VCS, where it moves a file in the tree?